js/tweek-client/src/TweekManagementClient/types.ts

```diff
@@ -65,6 +65,15 @@ export type ServiceDetails = {
   status: string;
 };
 
+export type Policy = {
+  group: string;
+  user: string;
+  contexts: { [identityId: string]: string };
+  object: string;
+  action: 'read' | 'write' | '*';
```

Comment:
can you please make `action` and `effect` enums?

Reply:
```const enums``` right?

Reply:
of course :)